apps/aecore/lib/aecore/governance/governance_constants.ex

```diff
@@ -29,6 +29,8 @@ defmodule Aecore.Governance.GovernanceConstants do
 
   @name_claim_burned_fee 3
 
+  @max_txs_per_block 10946
```

Comment:
This is some duplication. Max txs per block should be defined in GovernaneConstants OR in config. Not both.

Reply:
Forgot to delete it.